.github/ISSUE_TEMPLATE/crash.yml

```diff
@@ -0,0 +1,54 @@
+name: Crash report
+description: A hard crash of the interpreter, possibly with a core dump
+labels: ["type-crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar.
+          Exceptions unexpectedly raised from stdlib Python functions count as bugs rather than crashes.
```

Comment:
```suggestion
          Exceptions unexpectedly raised from Python code count as bugs rather than crashes.
```

Reply:
Hmm. Aren't most Python crashes caused by Python code in one way or another? I feel like "from stdlib Python functions" is more precise here (and it's the wording we have in the current issue template -- though that was also written by me 😄)

Reply:
My specific objection was to "stdlib" -- I wanted to use a less jargon-y term, but then I framed the divide as "crash": C code has gone spectacularly wrong; "bug": error in Python code. Perhaps instead we should frame this negatively, and say wtteo "If your issue involves a Python exception, or another problem in pure-Python code, it is a bug rather than a crash"

Reply:
Gotcha. I think the most common issue (or at least, something that _used_ to be a common issue before we added this note to the issue template) is that people report it as a crash if `ipaddress.foo()` raises `IndexError` (or whatever) -- so mentioning the standard library specifically is quite helpful here. I think I'll change it to "functions in the standard library" -- do you think that'll be okay?